src/styles/_dialogs.scss

```diff
@@ -25,6 +25,16 @@
     font-size: $lu_dialog_font_size;
     clear: right;
 
+    &::before {
```

Comment:
do give it more space for mouse leave, right?

Reply:
Yes, exactly. So that the menu doesn't close already at the border.